src/lib/crypto.js

```diff
@@ -9,8 +9,19 @@ export function hash(message) {
 }
 
 export function saltAndHash(password, salt) {
-    password = Buffer.from(password, 'hex');
-    salt = Buffer.from(password, 'hex');
```

Comment:
Oh! This was not correct at all. It should have just used the salt that was passed.

:clap: